docs/ug/generatingReports.md

```diff
@@ -18,6 +18,12 @@ Let's look at different ways to generate RepoSense reports.
 
 * **If you don't have Java on your computer or do not wish to run the executable on your computer**, some alternatives are provided in the [_Generating reports remotely_](#generating-reports-remotely) section below.
 
+<box type="info" seamless>
+
+RepoSense is built to analyze any type of git repo, remote or local. It works best when analyzing remote repositories hosted on GitHub, GitLab or BitBucket.
+For other types of repositories, features such as external links might not work properly.
```

Comment:
Currently, the links for local repositories and non GitHub, GitLab and BitBucket repos do not open at all which I think was the desired outcome we agreed on. Regarding the wording used in the UG here, it was wrote in this way in view of https://github.com/reposense/RepoSense/pull/1679 where local repositories might be able to support links. This might also change if we include a way for users to specify their own remote repository URL templates. 

However, I am fine to change it to `For other types of repositories, external links are disabled.` as these other features might not be coming so soon, if ever.

Reply:
Yup I think the wording is better now. #1679 seems to be more aligned with what I was thinking about, so I think we can leave this change to that PR.